Sources/Diagnostic/DiagnosticsFormatter.swift

```diff
@@ -68,13 +68,15 @@ public struct DiagnosticsFormatter {
     let sourceLines = sourceCode.components(separatedBy: "\n")
     guard let sourceLocation = sourceLocation else { return "" }
 
-    let spaceOffset = sourceLocation.column != 0 ? sourceLocation.column - 1 : 0
+    let sourceLine = sourceLines[sourceLocation.line - 1].replacingOccurrences(of: "\t", with: " ")
```

Comment:
Should this be double spaced?

Reply:
I'm only replacing tabs with single spaces as I couldn't think of an easy way to count the number of tabs as they are replaced so additional spaces can be added to the indicator.

Reply:
Ideally this should be rewritten so it finds where the diagnostic message should start on the line rather than indenting one space for each tab in the original diagnostic.